config/routes.rb

```diff
@@ -1,10 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  #sessions
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+  #static pages
   root "static_pages#home"
   get "static_pages/home"
   get "static_pages/help"
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  #sign up
```

Comment:
Should add enter line in each comment
```suggestion

  #sign up
```